### PR TITLE
Feature/pelagos 4037 create vue app for research group landing page

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: templates/ResearchGroup/index.html.twig [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{% extends custom_base_template | orTemplateIfNotExists('base.html.twig') %}
```
Comment (praneethpr) [inline templates/ResearchGroup/index.html.twig:1]: Change to base template

File: assets/js/research-group.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import Vue from "vue";
+import '../scss/bootstrap.scss';
+import '@fortawesome/fontawesome-free/css/all.min.css';
```
Comment (praneethpr) [inline assets/js/research-group.js:3]: Can remove this

File: assets/js/vue/ResearchGroupApp.vue [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+                    <PeopleTab :personResearchGroups="researchGroupData.personResearchGroups" />
+                </b-tab>
+                <b-tab title="Publications">
+                    <b-card-text>
+                        Alesia Ferguson, Helena Solo-Gabriele, Kristina Mena. 2020.
+                        Child specific and beach characteristic dataset. Distributed by: GRIIDC,
+                        Harte Research Institute, Texas A&M University-Corpus Christi. doi:10.7266/n7-rq3z-hq57
+                    </b-card-text>
+                </b-tab>
+            </b-tabs>
```
Comment (praneethpr) [inline assets/js/vue/ResearchGroupApp.vue:22]: Add v-else if research group is not found

File: assets/js/vue/components/research-group/DatasetsTab.vue [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+        props: {
+            datasets: {}
+        },
+        data() {
+            return {
+                datasetsRetrievedInfo: []
+            }
+        },
+        mounted() {
+            this.datasets.forEach(dataset => {
```
Comment (praneethpr) [inline assets/js/vue/components/research-group/DatasetsTab.vue:32]: Remove API calls, make citation into a virtual property

File: assets/js/vue/components/research-group/DatasetsTab.vue [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+                let loader = null;
+                let thisComponent = this;
+                const axiosInstance = axios.create({});
+                axiosInstance.interceptors.request.use(function (config) {
+                    loader = thisComponent.$loading.show({
+                        container: thisComponent.$refs.formContainer,
```
Comment (praneethpr) [inline assets/js/vue/components/research-group/DatasetsTab.vue:38]: Add ref variable name to the component

File: assets/js/vue/ResearchGroupApp.vue [hunk 6]
```diff
@@ -0,0 +1,91 @@
+<template>
+    <div class="container-xl" ref="formContainer">
+        <hr>
+        <b-card no-body class="main-card">
+            <b-tabs pills fill justified card v-if="showData" lazy vertical class="vh-100">
```
Comment (mickel1138) [inline assets/js/vue/ResearchGroupApp.vue:5]: Not a big fan of pills, but it's ok.

**Pull request merged**